vision/getting-started/image_editing.ipynb

```diff
@@ -1,710 +1,1247 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        exif = img_pil.getexif()
</pre><p>Could you also include a description of the maybe_rotate() function? Based on the function name alone, it's a bit unclear what this function would do and how it would know if the image should be rotated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/506/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Added more details to the comment above</p>